cupy/cuda/memory_hooks/line_profile.py

```diff
@@ -0,0 +1,171 @@
+from os import path
+import sys
+import traceback
+
+from cupy.cuda.memory_hook import MemoryHook
```

Comment:
Sorry, I more point.
Please do not use direct class import.
Use `from cupy.cuda import memory_hook`

Reply:
fixed